bgp/tests/local_tests/policy_test.go

```diff
@@ -352,9 +354,22 @@ func testAttrs(t *testing.T, routeTest *valpb.RouteTestCase, prevDUT, currDUT, n
 	// SetOrigin to artificially set this attribute, then remove this
 	// function and associated logic.
 	nonNilOrIGP := func(a *valpb.RibAttributes, rejected bool) *valpb.RibAttributes {
```

Comment:
This function was renamed `defaultAttrs` in https://github.com/openconfig/lemming/pull/410